src/snowflake/snowpark/modin/pandas/series_utils.py

```diff
@@ -1039,18 +1073,23 @@ def capitalize(self):
         return Series(query_compiler=self._query_compiler.str_capitalize())
 
     def swapcase(self):
+        ErrorMessage.not_implemented("Series.str.swapcase() is not implemented")
```

Comment:
one suggestion (no need to follow through): we could also have a @not_implemented decorator, that automatically take the function name and issue the string. Then all messages are formatted the same and you do not need to manually edit all of them :)

Reply:
That's a great idea! I tried it with the decorator raising the error but I don't think it executes as expected with tests. I think I will stick with what I have for now though, thanks!